BootloaderCorePkg/BootloaderCorePkg.dec

```diff
@@ -125,6 +125,8 @@
   gPlatformModuleTokenSpaceGuid.PcdAcpiTablesAddress      | 0xFF000000 | UINT32 | 0x20000110
   gPlatformModuleTokenSpaceGuid.PcdAcpiGnvsAddress        | 0xFF000000 | UINT32 | 0x20000112
   gPlatformModuleTokenSpaceGuid.PcdGraphicsVbtAddress     | 0xFF000000 | UINT32 | 0x20000113
+  gPlatformModuleTokenSpaceGuid.PcdSmramTsegBase          | 0xFF000000 | UINT32 | 0x20000120
+  gPlatformModuleTokenSpaceGuid.PcdSmramTsegSize          | 0x00000000 | UINT32 | 0x20000121
```

Comment:
Comment above mentions not to have default value of 0 here as it may cause the PCD to be put into the BSS section.

Reply:
That is good point.  
But it is OK to keep the PCD in BSS section as long as we don't do static patching using PatchFv.py because the symbol won't be generated in MAP file.   So I will update the comment to make it more clear.